supplementary/devices/index.rst

```diff
@@ -120,7 +122,7 @@ Devices
      - Navigation
      - Sensors
      - Other
-   * - :doc:`Samsung Galaxy S8+ <dream2lte>`
+   * - `Samsung Galaxy S8+ <dream2lte>`
      - dream2lte
```

Comment:
I've built it locally now to double check. Yes, you are removing the hyperlinks to all subpages. Please revert those changes.